fw/ctrl_832/mmc.c

```diff
@@ -309,6 +298,7 @@ unsigned long MMC_GetCapacity()
 			result|=CSDData[8]<<18;
 			result|=CSDData[9]<<10;
 			result+=1024;
+			sd_lbas=result;
```

Comment:
needs to be removed, I don't see it defined anywhere, and it isn't used in this function.